_docs/_latest/develop/branch-methodology.md

```diff
@@ -0,0 +1,43 @@
+---
+title: Branch Methodology
+order: 003
+---
+
+#  {{ page.title }}
+
+## Branches
+
+There are several important main branches in Forseti repo to know about,
+in order to correctly inspect the right code, or create a Pull Request (PR).
+
+* `dev`: development branch for Forseti 1.0.
+* `master`: latest stable code for Forseti 1.0.
+* `forsetisecurity.org`: documentations for Forseti 1.0 website.
+
+* `2.0-dev`: development branch for Forseti 2.0.
+* `2.0-master`: latest stable code for Forseti 2.0.
+* `2.0-forsetisecurity.org-dev`: documentations for Forseti 2.0 website.
+
+Besides these main branches, you will also see other active branches.
+Those branches will be other developer's work-in-progress, and can be
+generally disregarded.
+
+## Branching Methodology
+
+1. Know the Forseti version that you want to make the change.
+
+1. Depending on whether you want to make a code change or a documentation change,
```

Comment:
We could simplify this:

"Check out the appropriate development or documentation branch locally, sync to remove, and then create a local feature branch for your change."

But we should also use common terminology with the contributing page. https://github.com/GoogleCloudPlatform/forseti-security/blob/master/.github/CONTRIBUTING.md

Should we update that page or this one?

Reply:
The CONTRIBUTING.md is more correct, so we should follow that here.  Is there something specific that you would recommend to be changed here?